javaparser-testing/src/test/java/com/github/javaparser/utils/VisitorMapTest.java

```diff
@@ -18,52 +19,62 @@ public void normalEqualsDoesDeepCompare() {
+        Map<CompilationUnit, Integer> map = new HashMap<>();
+        map.put(x1, 1);
+        map.put(x2, 2);
+        assertEquals(1, map.size());
     }
 
     @Test
     public void objectIdentityEqualsDoesShallowCompare() {
         CompilationUnit x1 = JavaParser.parse("class X{}");
         CompilationUnit x2 = JavaParser.parse("class X{}");
 
-        Map<CompilationUnit, Integer> normalMap = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
-        normalMap.put(x1, 1);
-        normalMap.put(x2, 2);
-        assertEquals(2, normalMap.size());
+        Map<CompilationUnit, Integer> map = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
+        map.put(x1, 1);
+        map.put(x2, 2);
+        assertEquals(2, map.size());
     }
     
     @Test
     public void visitorMapGet(){
     	CompilationUnit x1 = JavaParser.parse("class X{}");
 
-        Map<CompilationUnit, Integer> normalMap = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
-        normalMap.put(x1, 1);
-        assertEquals(1, (int)normalMap.get(x1));
+        Map<CompilationUnit, Integer> map = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
+        map.put(x1, 1);
+        assertEquals(1, (int)map.get(x1));
     }
     
     @Test
     public void visitorMapContainsKey(){
     	CompilationUnit x1 = JavaParser.parse("class X{}");
 
-        Map<CompilationUnit, Integer> normalMap = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
-        normalMap.put(x1, 1);
-        assertTrue(normalMap.containsKey(x1));
+        Map<CompilationUnit, Integer> map = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
+        map.put(x1, 1);
+        assertTrue(map.containsKey(x1));
     }
     
     @Test
     public void visitorMapPutAll(){
     	CompilationUnit x1 = JavaParser.parse("class X{}");
     	CompilationUnit x2 = JavaParser.parse("class Y{}");
-    	Map<CompilationUnit, Integer> normalMap = new HashMap<>();
-    	normalMap.put(x1, 1);
-    	normalMap.put(x2, 2);
+    	Map<CompilationUnit, Integer> map = new HashMap<>();
+    	map.put(x1, 1);
+    	map.put(x2, 2);
     	Map<CompilationUnit, Integer> visitorMap = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
-        visitorMap.putAll(normalMap);
+        visitorMap.putAll(map);
         assertEquals(2, visitorMap.size());
     }
     
-
+    @Test
+    public void remove(){
+        CompilationUnit x1 = JavaParser.parse("class X{}");
+        VisitorMap<CompilationUnit, Integer> map = new VisitorMap<>(new ObjectIdentityHashCodeVisitor(), new ObjectIdentityEqualsVisitor());
+        map.put(x1, 1);
+        assertTrue(map.containsKey(x1));
+        
+        map.remove(x1);
+        
+        assertFalse(map.containsKey(x1));
+    }
 }
```

Comment:
newline :)

Reply:
I left it there for you to complain about :-)